src/molecule.py

```diff
@@ -1668,17 +1645,17 @@ def write(self,fnm=None,ftype=None,append=False,selection=None,**kwargs):
     #=====================================#
 
     def center_of_mass(self):
-        totMass = sum([PeriodicTable.get(self.elem[i], 0.0) for i in range(self.na)])
```

Comment:
The lines highlighted in red are newer, so the changes here need to be reverted, perhaps by merging the master branch into this branch.